scope_shared/scope/schemas/utils/enums.json

```diff
@@ -209,6 +209,22 @@
 			},
 			"required": ["Flag as safety risk", "Flag for discussion"]
 		},
+		"documentType": {
```

Comment:
These aren't really "documentType", they're just the set types?

Reply:
Fixed